android/java/res/drawable/custom_tip_cancel_selector.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape android:shape="rectangle"  >
+            <corners android:radius="20dip" />
+            <solid android:color="#AEB1C2"/>
```

Comment:
nit: use colour resources and remove dupe

Reply:
done